src/reducers/entities/roles.ts

```diff
@@ -1,45 +1,56 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-
 import {combineReducers} from 'redux';
 import {RoleTypes, UserTypes} from 'action_types';
 
 function pending(state = new Set(), action) {
     switch (action.type) {
     case RoleTypes.SET_PENDING_ROLES:
         return action.data;
+
     case UserTypes.LOGOUT_SUCCESS:
         return new Set();
+
     default:
         return state;
     }
 }
 
 function roles(state = {}, action) {
     switch (action.type) {
-    case RoleTypes.RECEIVED_ROLES: {
+    case RoleTypes.RECEIVED_ROLES:
+    {
         if (action.data) {
-            const nextState = {...state};
+            const nextState = {...state,
+            };
+
             for (const role of action.data) {
                 nextState[role.name] = role;
             }
+
             return nextState;
         }
 
         return state;
     }
-    case RoleTypes.ROLE_DELETED: {
+
+    case RoleTypes.ROLE_DELETED:
+    {
         if (action.data) {
-            const nextState = {...state};
+            const nextState = {...state,
+            };
             Reflect.deleteProperty(nextState, action.data.name);
             return nextState;
         }
 
         return state;
     }
-    case RoleTypes.RECEIVED_ROLE: {
+
+    case RoleTypes.RECEIVED_ROLE:
+    {
         if (action.data) {
-            const nextState = {...state};
+            const nextState = {...state,
+            };
```

Comment:
This instance of this notation sort of looks sort of sloppy. I think the application's convention is to have this on one line.

A lot of single-line -> multi-line and multi-line -> single-line changes make this PR bigger than necessary imo. Were a lot of these changes done with an automated tool?

Reply:
This is a result of automatic conversion. I've tried to fix most of the formatting issues, but some are still lingering, not sure how to auto-fix this stuff.